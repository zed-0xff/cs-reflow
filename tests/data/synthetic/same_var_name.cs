static void Main()
{
    {
        int i = 123;
        Console.WriteLine(i);
    }

    {
        int i = 123;
        Console.WriteLine(i);
    }
}
