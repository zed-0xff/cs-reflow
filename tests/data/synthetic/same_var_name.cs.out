static void Main()
{
    {
        int i = 123;
        Console.WriteLine(i);
    }

    {
        int i = 456;
        Console.WriteLine(i);
    }
}
